### PR TITLE
Update instructions for using rclone w/ Permanent

### PR DESCRIPTION
**Pull request by kfogel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,43 +37,82 @@ npm run dev
...
+Run **`rclone config`** and answer the questions it asks:
 
-At the end of the configuration you should have something like this:
+- Select option **`n`** (for "new") and make a remote name (e.g., "permanent").
+
+- At the **`Storage>`** prompt, set the storage protocol to SFTP, either by typing "sftp" at the prompt or by selecting the corresponding number from the displayed menu (the numbered entry may say "SSH/SFTP").
+
+- At the **`host>`** prompt, set the host to **`sftp.permanent.org`**.
+
+  If you want to run rclone against the dev or staging environment, then use `sftp.dev.permanent.org` or `sftp.staging.permanent.org` instead.  (You can also say `sftp.prod.permanent.org`, but it means the same thing as just `sftp.permanent.org`.)
```
Comment (Fenn-CS) [inline README.md:53]: > (You can also say `sftp.prod.permanent.org`, but it means the same thing as just `sftp.permanent.org`.)

I wonder if this is true.

Comment (kfogel) [reply README.md:53]: Well, I'm merely repeating a claim that was in the original.

Comment (kfogel) [reply README.md]: > I wonder if this is true.

It is not true -- I misread some material about dev and staging in the original text, and mistakenly applied it to prod as well.  Thanks for catching this!  I've pushed the fix (commit e1e92104712) to this branch.

**Review by Fenn-CS — APPROVED** [review]
Thanks for the upgrade @kfogel

**Comment by kfogel** [comment]
Merged!  Thanks, @Fenn-CS.

**Pull request merged**